Makefile

```diff
@@ -48,6 +48,17 @@ endif
 
 ETC_LOCALTIME := $(realpath /etc/localtime)
 
+HOST_ARCH ?= $(shell arch)
+ifeq ($(HOST_ARCH),x86_64)
+    DOCKER_PLATFORM := "linux/amd64"
+else ifeq ($(HOST_ARCH),arm64)
+	# We assume ARMv8 as the platform
```

Comment:
```suggestion
```

Maybe remove this comment since we're not specifying it explicitly anymore? I think before I had `linux/arm64/v8` or something.

Reply:
Good point. arm64 is automatically v8 (v7 is 32-only afaik).

Reply:
I guess it could be v9

Reply:
hm, actually, this also needs to know about `i386` on Intel Macs.